docker-compose.yml

```
@@ -1,50 +1,59 @@
     build:
-      context: ./src
+      context: .  # Build context is the project root.
+      dockerfile: web.Dockerfile # Uses web.Dockerfile for building the Flask app image.
     ports:
-      - "5000:5000"
+      - "5000:5000" # Maps port 5000 of the host to port 5000 of the container (Flask default).
     volumes:
+      # Mounts the local src/web directory into the container for live code reloading during development.
       - type: bind
-        source: ./src/web
+        source: ./src/web 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)